presto-product-tests/src/main/resources/sql-tests/datasets/orc_varchar_dictionary.ddl

```diff
@@ -0,0 +1,6 @@
+-- type: hive
+CREATE EXTERNAL TABLE %NAME%(
```

Comment:
Maybe create another table using the same file but with `VARCHAR(4)` and assert that the value is `colu`